code/drasil-example/Drasil/GlassBR/Unitals.hs

```diff
@@ -229,10 +237,11 @@ is_safeLR      = vc "is_safeLR"        (nounPhraseSP $ "variable that is assigne
 lDurFac       = vc'' (loadDurFactor) (Atomic "LDF") Real
 
 loadSF        = vc'' (lShareFac) (Atomic "LSF") Natural
+----------------------Should be moved to UnitryChunckSection-----------------------------------
```

Comment:
Please delete both the comment and the commented out code.

Reply:
@JacquesCarette it was addressed with 0bd1b19.